<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location-Allocation</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Trace"/>
    <addaction name="actionOpen_Shapefile"/>
    <addaction name="actionSet_projection"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Grid"/>
    <addaction name="actionExport_Trace"/>
    <addaction name="actionExport_ONETrace"/>
    <addaction name="actionExport_Trace_grid"/>
    <addaction name="actionExport_Trace_txt"/>
    <addaction name="separator"/>
    <addaction name="actionSpatial_Stats"/>
    <addaction name="actionShow_intermediate_points"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen_Trace">
   <property name="text">
    <string>Open Trace</string>
   </property>
  </action>
  <action name="actionOpen_Shapefile">
   <property name="text">
    <string>Open Shapefile</string>
   </property>
  </action>
  <action name="actionSet_projection">
   <property name="text">
    <string>Set projection</string>
   </property>
  </action>
  <action name="actionShow_layer">
   <property name="text">
    <string>Show layers</string>
   </property>
  </action>
  <action name="actionAdd_Grid">
   <property name="text">
    <string>Add Grid</string>
   </property>
  </action>
  <action name="actionCompute_allocation">
   <property name="text">
    <string>Compute allocation</string>
   </property>
  </action>
  <action name="actionExport_Trace">
   <property name="text">
    <string>Export Trace</string>
   </property>
  </action>
  <action name="actionExport_ONETrace">
   <property name="text">
    <string>Export ONE Trace</string>
   </property>
  </action>
  <action name="actionExport_facilities">
   <property name="text">
    <string>Export facilities</string>
   </property>
  </action>
  <action name="actionSpatial_Stats">
   <property name="text">
    <string>Spatial Stats</string>
   </property>
  </action>
  <action name="actionExport_Trace_txt">
   <property name="text">
    <string>Export Trace (txt)</string>
   </property>
  </action>
  <action name="actionExport_Trace_grid">
   <property name="text">
    <string>Export Trace (grid)</string>
   </property>
  </action>
  <action name="actionShow_intermediate_points">
   <property name="text">
    <string>Show intermediate points</string>
   </property>
  </action>
  <action name="actionLaunch_REST_server">
   <property name="text">
    <string>Launch REST server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
